extensions/ql-vscode/src/model-editor/model-alerts/model-alerts-view.ts

```diff
@@ -195,4 +204,15 @@ export class ModelAlertsView extends AbstractWebview<
       method,
     );
   }
+
+  public async revealMethod(method: ModeledMethod): Promise<void> {
```

Comment:
Does this need to be public?

Reply:
Nope 😄 Will make it private! Thanks for spotting that 👀